README.md

```diff
@@ -127,37 +108,25 @@ TODO
 
 TODO
 
-### Additional TorchScript end-to-end tests with heavy dependencies
 
-Some of the Torchscript end-to-end tests require additional dependencies which
-don't make sense to include as part of the default torch-mlir setup.
```

Comment:
can you move this into a comment in build_tools/torchscript_e2e_heavydep_tests/generate_serialized_tests.sh -- it's the only load-bearing documentation for this feature, so I don't want to outright delete it.

Reply:
Actually build_tools/torchscript_e2e_heavydep_tests/README.md would be better

Reply:
Will do in a follow on commit